### ISSUE TITLE
CSS transform makes element invisible

### ISSUE THREAD

**Issue by annulen** [annulen/webkit#30]
E.g. LayoutTests/fast/css/transform-default-parameter.html

**Comment by annulen** [comment]
Here is simplified test case:
```html
<html>
  <head>
    <style type="text/css">
      #box1 {
        background-color: blue;
        -webkit-transform: translateZ(0)
      }
    </style>
  </head>
  <body>
    <div id="box1">
      This line should be visible.
    </div>
  </body>
</html>
```

**Comment by annulen** [comment]
This bug was present because we were going for mode with disabled accelerated compositing, but  WebCore was not informed about this setting and assumed it to be enabled.

**Comment by annulen** [comment]
Fixes: 49df131cc3cfb5936f98d698e0e07506feed85c0, e60e43d73f2c10ed17c2cedda679a3a901d4b0e5
